app/src/main/java/com/kms/cura/view/fragment/MessageListFragment.java

```diff
@@ -0,0 +1,231 @@
+            messageEntities.add(entity6);
+            messageEntities.add(entity7);
+            messageEntities.add(entity8);
+            messageEntities.add(entity9);
+            messageEntities.add(entity10);
+            messageEntities.add(entity11);
+            messageEntities.add(entity12);
+            messageEntities.add(entity13);
+            messageThreadEntities = MessageThreadEntity.getMessageThreadFromList(patientUserEntity, messageEntities);
+        } else {
+            user = new PatientUserEntity("1", "Peter Pan");
+            user2 = new PatientUserEntity("2", "Doraemon");
+            DoctorUserEntity doctorUserEntity = (DoctorUserEntity) CurrentUserProfile.getInstance().getEntity();
+            MessageEntity entity1 = new MessageEntity(user, doctorUserEntity, timestamp, "Hello from the other side");
+            MessageEntity entity4 = new MessageEntity(user2, doctorUserEntity, timestamp, "Hello");
+            timestamp.setTime(Calendar.getInstance().getTimeInMillis() + 3000);
+            MessageEntity entity2 = new MessageEntity(doctorUserEntity, user, timestamp, "Hi");
+            MessageEntity entity5 = new MessageEntity(doctorUserEntity, user2, timestamp, "Hi");
+            timestamp.setTime(Calendar.getInstance().getTimeInMillis() + 6000);
+            MessageEntity entity3 = new MessageEntity(user, doctorUserEntity, timestamp, "Bye");
+            MessageEntity entity6 = new MessageEntity(user2, doctorUserEntity, timestamp, "Bye");
+            messageEntities.add(entity1);
+            messageEntities.add(entity2);
+            messageEntities.add(entity3);
+            messageEntities.add(entity4);
+            messageEntities.add(entity5);
+            messageEntities.add(entity6);
+            messageThreadEntities = MessageThreadEntity.getMessageThreadFromList(doctorUserEntity, messageEntities);
+        }
+
+    }
+
+    @Override
+    public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+        if (multiMode) {
+            setSelection(view, position);
+            if (adapter.getSelectedCount() == 0) {
+                resetNavigationDrawer();
+            } else {
+                modifyToolbar(adapter.getSelectedCount() + " selected");
+            }
+        } else {
+            Toast.makeText(getActivity(), messageThreadEntities.get(position).getConversationName(CurrentUserProfile.getInstance().getEntity()), Toast.LENGTH_SHORT).show();
+        }
+    }
+
+    private void setSelection(View view, int position) {
+        if (!adapter.isSelected(position)) {
+            setLayoutColor(view, R.color.grey);
+            adapter.setSelected(position, true);
+        } else {
+            setLayoutColor(view, R.color.white);
+            adapter.setSelected(position, false);
+        }
+    }
+
+    @Override
+    public boolean onItemLongClick(AdapterView<?> parent, View view, int position, long id) {
+        if (!multiMode) {
```

Comment:
Should check if (multiMode) to return false right away to avoid lots of code in if block